sdk/spring/azure-spring-cloud-stream-binder-servicebus-queue/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ This release is compatible with Spring Boot 2.5.0 - 2.5.3 and Spring Cloud 2020.
 ### Dependency Upgrades
 - Upgrade to [spring-boot-dependencies:2.5.3](https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot-dependencies/2.5.3/spring-boot-dependencies-2.5.3.pom).
 
+### Breaking Changes
+- Remove the default logic for automatically adding partition key for each message. ([#23135](https://github.com/Azure/azure-sdk-for-java/pull/23135))
 
```

Comment:
The content should also inclde 
1. override paritionkey when session id is set
2. adjust the order of different partitionkey header.